pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```diff
@@ -1488,6 +1488,31 @@ spec:
                             type: object
                         type: object
                       type: array
+                    icmp:
+                      description: "ICMP is a list of ICMP rule identified by type
+                        number which the endpoint subject to the rule is allowed to
+                        receive connections on. \n Example: Any endpoint with the
+                        label \"app=httpd\" can only accept incoming type 8 ICMP connections."
+                      items:
+                        description: ICMPRule is a list of ICMP-type.
+                        properties:
+                          types:
```

Comment:
Yes, I don't think `types` is the best name.
However, currently we don't support ICMP codes or other header's field because there is no usecase, and also we don't have any plan to support them. Thus I think `types` is ok.

Or, we can name like `HeaderFields`, `attributes` or `TypeCodes`?

What do you think?

Reply:
`fields` is used in the [RFC](https://datatracker.ietf.org/doc/html/rfc792), so maybe that's a good option?

Reply:
I think it's good idea :)

Reply:
> we don't have any plan to support them.

We may support codes in the future.